migrations/archive/2019/20190731_outage_repair_groups.js

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable no-console */
+const MIGRATION_NAME = '20190731_outage_repair_groups';
+import { model as Group } from '../../website/server/models/group';
+import { model as User } from '../../website/server/models/user';
+import forEach from 'lodash/forEach';
+
+const progressCount = 1000;
+let count = 0;
+
+async function updateGroup (group) {
+  count++;
+
+  const set = {};
+
+  if (group.type === 'guild') {
+    set.memberCount = await User.countDocuments({guilds: group._id});
+  } else {
+    set.memberCount = await User.countDocuments({party._id: group._id});
+  }
+
+  if (group.quest && group.quest.key && group.quest.members) {
+    forEach(group.quest.members, async (status, memberId) => {
+      if (status === null) await User.update({_id: memberId, {$set: {
+        'party.quest.RSVPNeeded': true,
+        'party.quest.key': group.quest.key,
+      }}});
+      if (status === true) await User.update({_id: memberId, {$set: {
+        'party.quest.key': group.quest.key,
+      }}})
```

Comment:
It might be worth setting `party.quest.RSVPNeeded` to false for both `(status === true)` and `(status === false)`. We see a trickle of people needing RSVPNeeded to be turned off (not necessarily from the recent group troubles, just as a general thing; usually it's because they're no longer in a party but not always).

Reply:
Updated!